Update Sessions page with new filters and columns. (#15080)
cockroachdb/docs: _includes/v22.2/ui/sessions.md

<img src="{{ 'images/v22.2/ui-session-filter.png' | relative_url }}" alt="Session filter" />

- To filter by [application]({{ link_prefix }}connection-parameters.html#additional-connection-parameters), select **App** and choose one or more applications.

    - Queries from the SQL shell are displayed under the `$ cockroach` app.

>     - If you haven't set `application_name` in a client connection string, it appears as `unset`.

Reviewer: ```suggestion
    - If you have not set `application_name` in a client connection string, it appears as `unset`.
```

> - To filter by username or session status, select **Username** or **Session Status** and choose one or more checkboxes.

Reviewer 2: Nit: User name 

<img width="146" alt="image" src="https://user-images.githubusercontent.com/81193618/189795310-94cf0d0b-eef9-4645-8be4-f6690ad6ff4e.png">

Author: @kevin-v-ngo The checkbox in the filter is labeled **Username**. See the screenshot ui-session-filter.png in this PR.

Author: @kevin-v-ngo  I filed this: https://github.com/cockroachdb/cockroach/issues/87960

> - To filter by username or session status, select **Username** or **Session Status** and choose one or more checkboxes.

Reviewer: Nit: It feels a little weird to "choose" a checkbox, but I grant that "check" may be redundant. Maybe "fill" or "click"?

```suggestion
- To filter by username or session status, select **Username** or **Session Status** and check one or more checkboxes.
```

- To filter by session duration, specify the session time and unit.

Click <img src="{{ 'images/common/ui-columns-button.png' | relative_url }}" alt="Column selector" /> to select the columns to display in the table.

The following properties are displayed for each session:

Column | Description
--------- | -----------
Session Start Time (UTC) | The timestamp at which the session started.
Session Duration | The amount of time the session has been open.
Session Active Duration | The amount of time transactions were executing while the session was open.

> Status  | The status of the session: Active, Closed, or Idle. A session is Active if it has an open explicit or implicit transaction (individual SQL statement) with a statement that is actively running or waiting to acquire a lock. A session is Closed if it has closed the connection to CockroachDB. A session is Idle if it is not executing a statement.

Reviewer: If "Active", "Closed" and "Idle" are the actual status names, I think they should be in backticks.

> Status  | The status of the session: Active, Closed, or Idle. A session is Active if it has an open explicit or implicit transaction (individual SQL statement) with a statement that is actively running or waiting to acquire a lock. A session is Closed if it has closed the connection to CockroachDB. A session is Idle if it is not executing a statement.

Author: ```suggestion
Status  | The status of the session: `Active`, `Closed`, or `Idle`. A session is `Active` if it has an open explicit or implicit transaction (individual SQL statement) with a statement that is actively running or waiting to acquire a lock. A session is `Closed` if it has closed the connection to CockroachDB. A session is `Idle` if it is not executing a statement.
```
